pkgs/applications/audio/mmtc/default.nix

```diff
@@ -0,0 +1,40 @@
+{ callPackage, fetchFromGitHub, rustPlatform, stdenv }:
+
+let
+  rust = (let
+    mozilla = fetchFromGitHub {
+      owner = "mozilla";
+      repo = "nixpkgs-mozilla";
```

Comment:
I don't think this is acceptable in nixpkgs.

Reply:
> I don't think this is acceptable in nixpkgs.

I guess that's why, I will close this then